cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -350,6 +364,54 @@ protected void onCreate(Bundle savedInstanceState) {
     videoSurfaceView = new VideoSurfaceView(this);
     addContentView(
         videoSurfaceView, new LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT));
+
+    initializeJavaBridge();
+  }
+
+  /**
+   * Initializes the Java Bridge to allow communication between Java and JavaScript.
+   * This method injects Java objects into the WebView and loads corresponding JavaScript code.
+   */
+  private void initializeJavaBridge() {
+
+    WebContents webContents = getActiveWebContents();
+    if (webContents == null) {
```

Comment:
Is it possible that this produces an infinite recursion loop here ? Should there a be a limit / counter that eventually raises an exception ?

Reply:
Not likely, it waits for the shell to be created, once it is created, webContents will not be null.